libs/dkg/include/dkg/rbc_messages.hpp

```diff
@@ -46,14 +46,27 @@ class RBCMessage
 public:
   enum class MessageType : uint8_t
   {
-    RBROADCAST,
-    RECHO,
-    RREADY,
-    RREQUEST,
-    RANSWER
+    R_BROADCAST,
+    R_ECHO,
+    R_READY,
+    R_REQUEST,
+    R_ANSWER,
+    R_SEND
```

Comment:
R_SEND is not needed. This was in the consensus repo and is what is called R_BROADCAST here.

Reply:
OK, will change accordingly.